Doc/library/datetime.rst

```diff
@@ -14,40 +14,59 @@
-both simple and complex ways.  While date and time arithmetic is supported, the
-focus of the implementation is on efficient attribute extraction for output
-formatting and manipulation. For related functionality, see also the
-:mod:`time` and :mod:`calendar` modules.
+The :mod:`datetime` module supplies classes for manipulating dates and times.
 
-There are two kinds of date and time objects: "naive" and "aware".
+While date and time arithmetic is supported, the focus of the implementation is
+on efficient attribute extraction for output formatting and manipulation.
 
-An aware object has sufficient knowledge of applicable algorithmic and
+.. seealso::
+
+   Module :mod:`calendar`
+      General calendar related functions.
+
+   Module :mod:`time`
+      Time access and conversions.
+
+   Package `dateutil <https://dateutil.readthedocs.io/en/stable/>`_
+      Third-party library with expanded time zone and parsing support.
+
+.. _datetime-naive-aware:
+
+Naive vs Aware Objects
+----------------------
+
+There are two kinds of date and time objects: "naive" and "aware."
+
+An **aware** object has sufficient knowledge of applicable algorithmic and
 political time adjustments, such as time zone and daylight saving time
-information, to locate itself relative to other aware objects.  An aware object
+information, to locate itself relative to other aware objects. An aware object
 is used to represent a specific moment in time that is not open to
 interpretation [#]_.
 
-A naive object does not contain enough information to unambiguously locate
-itself relative to other date/time objects.  Whether a naive object represents
+A **naive** object does not contain enough information to unambiguously locate
+itself relative to other date/time objects. Whether a naive object represents
 Coordinated Universal Time (UTC), local time, or time in some other timezone is
 purely up to the program, just like it is up to the program whether a
-particular number represents metres, miles, or mass.  Naive objects are easy to
+particular number represents metres, miles, or mass. Naive objects are easy to
 understand and to work with, at the cost of ignoring some aspects of reality.
 
 For applications requiring aware objects, :class:`.datetime` and :class:`.time`
 objects have an optional time zone information attribute, :attr:`!tzinfo`, that
 can be set to an instance of a subclass of the abstract :class:`tzinfo` class.
 These :class:`tzinfo` objects capture information about the offset from UTC
-time, the time zone name, and whether Daylight Saving Time is in effect.  Note
-that only one concrete :class:`tzinfo` class, the :class:`timezone` class, is
-supplied by the :mod:`datetime` module.  The :class:`timezone` class can
+time, the time zone name, and whether daylight saving time is in effect.
+
+Only one concrete :class:`tzinfo` class, the :class:`timezone` class, is
+supplied by the :mod:`datetime` module. The :class:`timezone` class can
```

Comment:
had to re-read that 3 time. I get it now, and make perfect sens... but I had to reread it. Maybe split there after `module.` again ?

Reply:
Addressed in 36c564c874.